### DISCUSSION TITLE
rgument XX: Got invalid value '1' on prisma.XX. Provided String, expected Int

### DISCUSSION THREAD

**Discussion by shaitalatgoola** [prisma/prisma#11672] [Q&A]
I send int but it turns out to be string
```
Argument lang: Got invalid value '1' on prisma.createOnedoc_files. Provided String, expected Int
    @IsNumber()
    @IsNotEmpty()
    @Type(() => Number)
    lang: number;
```

**Comment by ASR4** [comment]
Maybe this helps: https://github.com/prisma/prisma/issues/2580

**Comment by pantharshit00** [comment]
Hello @shaitalatgoola , our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

If this still an issue, can you please share you schema and the client call? You might need to coerce String to Int using `parseInt()` if you have int ids and passing a string in there.

**Reply by vladi-stevanovic** [reply]
👋 Hello @shaitalatgoola I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Reply by shaitalatgoola** [reply]
Hi, I dont know, I was long ago.
I guess you can close the issue

**Reply by vladi-stevanovic** [reply]
Ok, thank you for letting us know - if anything else pop-ups don't hesitate to open a new GH Discussion or post on our [Slack community](https://slack.prisma.io/)!

**Reply by myyrez** [reply]
wow you literally just saved me!!!!! Hours trying to solve something that just needed a simple parseInt... I guess that's programmer's life right lol

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
